string name
string resources
int64 duration
string precedence
string location_X
string location_Y